### PR TITLE
Fix display of mentor-favs within ratings

### PR DESCRIPTION
**Pull request by klappradla**
Updated the application table and application-rating view to show a
heart for each mentor-fav (updated the csv export to contain the
respective "true" and "false" values as well).

closes rails-girls-summer-of-code/selection_process#57

### CODE REVIEW

**Comment by beanieboi** [comment]
looks good to me, can't wait to move the ApplicationData into the database and have real data types 👯‍♂️

**Comment by klappradla** [comment]
Did you find the time to take a look at this already @mkalininait ? If it's fine, I'd actually straight merge it together with #732 _(which does basically the same just as a "non-fix")_ and run the rake task to clean up the DB (cc @carpodaster ).

File: app/views/rating/applications/_application.html.slim [hunk 1]
```diff
@@ -18,8 +18,8 @@ tr
   td.text-left = application.flags.map { |f| content_tag(:span, f, class: 'label label-default') }.join('<br>').html_safe
   td.text-center = icon('check') if application.application_data['signed_off_at_project1']
   td.text-center = icon('check') if application.application_data['signed_off_at_project2']
-  td.text-center = icon('check') if application.application_data['mentor_fav_project1']
-  td.text-center = icon('check') if application.application_data['mentor_fav_project2']
+  td.text-center = icon('heart') if application.application_data['mentor_fav_project1'] == 'true'
```
Comment (carpodaster) [inline app/views/rating/applications/_application.html.slim]: What about `[…].to_s == 'true'`? That way we also support real booleans.

Comment (klappradla) [inline app/views/rating/applications/_application.html.slim]: Sound defensive and *"future-proof"* love this 💚 
I have it in several places, will update it.

Does it need to be within this PR? It does not directly impact rating, so I could also do it a day later...?

Comment (carpodaster) [inline app/views/rating/applications/_application.html.slim]: We can do it later, of course!

Comment (klappradla) [inline app/views/rating/applications/_application.html.slim]: Ok. Will take care of merging and adding the respective task tonight. Nice, thanx 🙏

**Comment by mkalininait** [comment]
Tested locally. Looks good to me, thank you @klappradla!

File: app/views/rating/applications/_application.html.slim (hunk 1 above)
Comment (klappradla) [inline app/views/rating/applications/_application.html.slim]: Ok, already addressed it when merging `master` into #732 - it was only a few places where it made sense.

**Pull request merged**